### PR TITLE
[WNMGDS-2375] Fix privacy settings content in 6.1

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

WNMGDS-2375

During the process of cherry-picking our footer and privacy-settings work to v6.x, the order of two commits got flipped, resulting in the outdated privacy-settings content being saved in the [`release/6.1` branch](https://github.com/CMSgov/design-system/commits/release/6.1). [The content updates commit](https://github.com/CMSgov/design-system/pull/2453) was cherry-picked before [the migration of the privacy settings dialog to core](https://github.com/CMSgov/design-system/pull/2422). This PR fixes it by pulling that updated content from `main`.

## How to test

`yarn clean && yarn install && yarn storybook` and view the healthcare footer's privacy settings

Hopefully by making this a PR, we'll get some solid testing from our CI checks.

### CODE REVIEW

File: tests/browser/story-screenshots.test.ts-snapshots/components-pagination--default-core-chromium-forced-colors-linux.png
Comment (zarahzachz) [inline tests/browser/story-screenshots.test.ts-snapshots/components-pagination--default-core-chromium-forced-colors-linux.png:1]: ... is the padding supposed to change on pagination?

Comment (pwolfert) [reply tests/browser/story-screenshots.test.ts-snapshots/components-pagination--default-core-chromium-forced-colors-linux.png:1]: I think it was just the VRTs that didn't get updated on the release branch

File: tests/browser/story-screenshots.test.ts-snapshots/healthcare-logo--spanish-logo-healthcare-webkit-linux.png
Comment (zarahzachz) [inline tests/browser/story-screenshots.test.ts-snapshots/healthcare-logo--spanish-logo-healthcare-webkit-linux.png:1]: The spanish logos are now in english

File: tests/browser/story-screenshots.test.ts-snapshots/components-pagination--default-core-chromium-forced-colors-linux.png
Comment (pwolfert) [reply tests/browser/story-screenshots.test.ts-snapshots/components-pagination--default-core-chromium-forced-colors-linux.png:1]: The new version of these reference images is better, so I don't think it is a sign of a problem

File: tests/browser/story-screenshots.test.ts-snapshots/healthcare-footer--basic-footer-healthcare-chromium-linux.png
Comment (zarahzachz) [inline tests/browser/story-screenshots.test.ts-snapshots/healthcare-footer--basic-footer-healthcare-chromium-linux.png:1]: More padding changes

File: tests/browser/story-screenshots.test.ts-snapshots/healthcare-logo--spanish-logo-healthcare-webkit-linux.png
Comment (pwolfert) [reply tests/browser/story-screenshots.test.ts-snapshots/healthcare-logo--spanish-logo-healthcare-webkit-linux.png:1]: See this commit note: https://github.com/CMSgov/design-system/pull/2527/commits/c27eb21920a3a2acb917cf5fe16760459378b3ac

It's just a broken story

File: tests/browser/story-screenshots.test.ts-snapshots/healthcare-footer--basic-footer-healthcare-chromium-linux.png
Comment (pwolfert) [reply tests/browser/story-screenshots.test.ts-snapshots/healthcare-footer--basic-footer-healthcare-chromium-linux.png:1]: This is actually a fix

**Comment by pwolfert** [comment]
Back when this was first posted, we were told to hold off. At this point, we know that we're going to be removing the privacy settings modal entirely in favor of the analytics team owning that dialog. Therefore, this can be closed.

**Pull request closed without merging**